source/project-stories/reed-learning.erb

```diff
@@ -164,11 +174,20 @@ related:
       <p class="highlight__large-text"></p>
       <p class="highlight__small-text"><%= current_page.data.what_we_did_4.text %></p>
     </div>
-  </section>
+  </section> -->
```

Comment:
Don't comment things out, just delete them. If we need to refer to the code later we can just look back in the git history. This is especially important for html where people can look at this by inspecting the page source.